backend/Tasque.Core.WebAPI/appsettings.json

```diff
@@ -29,6 +27,8 @@
     "Host": "",
     "ConfirmationEndpoint": "",
     "PasswordResetEndpoint": "",
+    "SenderEmail": "",
+    "SenderName": "",
```

Comment:
If you change configuration, that may affect deployment, please, ping me, so I willl change it on server)